data/json/overmap/overmap_special/specials.json

```diff
@@ -827,7 +827,7 @@
     "connections": [ { "point": [ 0, -1, 0 ], "terrain": "road", "connection": "local_road" } ],
     "locations": [ "field", "forest_without_trail" ],
     "city_distance": [ 10, -1 ],
-    "occurrences": [ 10, 100 ],
+    "occurrences": [ 1, 10000 ],
```

Comment:
This is somewhere between too much and not enough.  Do we even want these around anymore?

Reply:
Well answered my own question.  Probably no but excising these from the game is probably too big to fold into this PR.